src/components/Menu/MenuItem/MenuItem.scss

```diff
@@ -0,0 +1,267 @@
+.menu-item:focus {
+  outline: none;
+}
+
+// MOBILE
+@media screen and (max-width: 1343px) {
+  $menu-item__first-position-left: 28px;
+
+  @for $i from 1 through 6 {
+    .menu-item#{$i} { left: $menu-item__first-position-left + (($i - 1) * 58px); }
+  }
+
+  .menu-item {
+    background-color: transparent;
+    border: none;
+    border-radius: 18px;
+    padding: 0;
+    position: absolute;
+    width: 36px;
+    transition: left 0s;
```

Comment:
Would this not be the same transition if you drop this line? `0s` indicates it for me

Reply:
Deleted it because made indeed no difference ✅